manifests/params.pp

```diff
@@ -5,10 +5,33 @@
     fail('This module only works on Debian or derivatives like Ubuntu')
   }
 
+  case $::lsbdistid {
+    'Debian','Raspbian': {
+      $package_version_greater_1_point_0 = '10'
+    }
+    'Ubuntu','Neon': {
+      $package_version_greater_1_point_0 = '18.04'
+    }
+    'LinuxMint': {
+      $package_version_greater_1_point_0 = '19'
+    }
+    default: {
+      warning('Can not assume unattended-upgrades package version')
```

Comment:
Probably good to define the variable. The default could be either `'0'` since the code can work with it even if not supported or very high so it's hidden. No real preference from my side.